src/icon-manifest/supportedExtensions.ts

```diff
@@ -579,6 +579,13 @@ export const extensions: IFileCollection = {
       languages: [languages.cabal],
       format: FileFormat.svg,
     },
+    {
+      icon: 'caddy',
+      extensions: ['caddyfile'],
```

Comment:
Since you added `languages`, `extensions` assignment is redundant.

Reply:
That wouldn't drop support for those without the 3rd party extension?

Reply:
Yes and that's our policy.